pflake8/__init__.py

```diff
@@ -51,16 +62,46 @@ class DivertingSafeConfigParser(ConfigParserTomlMixin, configparser.SafeConfigPa
     pass
 
 
-class ModifiedConfigFileFinder(flake8.options.config.ConfigFileFinder):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.project_filenames = self.project_filenames + ('pyproject.toml',)
+# Copied from flake8 version 5.0.2.
```

Comment:
Could we just wrap the function call instead of rewriting it?

I think flake8-pyproject did that; but I think this implementation does a better job of finding enclosing folders that contain a pyproject.toml file.

Reply:
Hey @schinckel the current implementation on `flake8` is a method with hardcoded list of candidate filenames.
We could do something like:
```
def _find_config_file(path: str) -> Optional[str]:
    result = flake8.options.config._find_config_file(path)
    if result is None:
        result = ...
    return result
    ```
But... without extracting methods from original `_find_config_file` it is basically the same method with a different list containing only ours `pyproject.toml`. I wrote a patch about extracting this candidate filename list, but it wasn't accepted.
In what way do you want to wrap the function call?
I don't want to change the logic of the original function.

Reply:
Yeah, that was what I was thinking about: but we'd possibly need to run our finder first.

Reply:
The finder code will be the same, equals duplication again.